get-new-tool-versions/get-new-tool-versions.ps1

```diff
@@ -29,7 +29,11 @@ if ($ToolName -eq "Xamarin") {
 if ($VersionsToBuild) {
     $availableVersions = $VersionsToBuild -join $joinChars
     Write-Host "The following versions are available to build:`n${availableVersions}"
-    Write-Host "##vso[task.setvariable variable=TOOL_VERSIONS;isOutput=true]${availableVersions}"
+    if ($ToolName -eq "Go") {
```

Comment:
Will we finally move all tools that use this file to new approach?
If yes, can we switch all of them together?

Reply:
Updated it to work with all tools